smartypants.go

```diff
@@ -398,13 +400,14 @@ func NewSmartypantsRenderer(flags Extensions) *SPRenderer {
 	return &r
 }
 
-func (sr *SPRenderer) Process(text []byte) []byte {
+// Process is the entry point of the Smartypants renderer.
+func (smrt *SPRenderer) Process(text []byte) []byte {
```

Comment:
Ah, I see. In that case I'd recommend going with `sr` or even just `r` since `smrt` is a very long receiver name and I don't think we should keep it only because it's already used in more places. No harm in a bigger diff if it makes the code better.